comictaggerlib/issueidentifier.py

```diff
@@ -229,10 +229,10 @@ def get_search_keys(self) -> SearchKeys:
 
         # see if the archive has any useful meta data for searching with
         try:
-            if ca.has_cix():
-                internal_metadata = ca.read_cix()
-            else:
-                internal_metadata = ca.read_cbi()
+            for style in metadata_styles:
+                internal_metadata = ca.read_metadata(style)
+                if not internal_metadata.is_empty:
+                    break
```

Comment:
It doesn't seem like there is a condition where `only_use_additional_meta_data` is ever `False` so this will be reached?

It also breaks on the first found metadata. Maybe not for this PR but part of the overlay/overwrite etc. work it should check each type if it's there? Leave a TODO?

Reply:
Yeah, I plan on removing the entire option later during a re-organization of the issue identifier.
I changed ComicArchiver to return an empty GenericMetadata if the archive doesn't have the requested metadata style, that way it stays consistent with the previous code.